### PR TITLE
Prepare for v8/v9 split

### PR DESCRIPTION
**Pull request by wincent**
This is a draft/RFC at this point, based on what I suggested in: https://github.com/liferay/liferay-themes-sdk/issues/184

I'll add more commits to this as I go. Some of the commits in here could be split off and merged independently of whether we decide to what #184 proposes, but for now I am just going to accumulate changes here so as to keep the number of PRs manageable.

These are targeting the "master" branch. I created an "8.x-stable" branch that won't hurt to have around in any case.

### CODE REVIEW

File: packages/liferay-theme-tasks/lib/upgrade/7.0/upgrade.js [hunk 1]
```diff
@@ -73,11 +73,11 @@ module.exports = function(options) {
 					patterns: [
 						{
 							match: /(DTD Look and Feel )\d(?:\.\d+)+(\/\/EN)/g,
-							replacement: '$17.0.0$2',
+							replacement: '$17.1.0$2',
 						},
 						{
 							match: /(liferay-look-and-feel_)\d(?:_\d+)+(\.dtd)/g,
-							replacement: '$17_0_0$2',
+							replacement: '$17_1_0$2',
```
Comment (wincent) [inline packages/liferay-theme-tasks/lib/upgrade/7.0/upgrade.js]: This is the bit I was referring to in the commit message where I said we weren't actually updating the version numbers.

The 7.1 DTD actually exists, as you can see here: https://github.com/liferay/liferay-portal/blob/master/definitions/liferay-look-and-feel_7_1_0.dtd

**Comment by wincent** [comment]
Still a work-in-progress, but adding you as a reviewer @julien in case you want to take a sneak peak at how this is evolving.

**Comment by julien** [comment]
@wincent I definetly will have a look, thanks

**Comment by wincent** [comment]
Still more that can be done here but removing "draft" status because I think it's pretty close now.

**Comment by wincent** [comment]
Just adding a reminder here for myself that https://github.com/liferay/liferay-js-themes-toolkit/pull/206 removes 7.2 support from the master branch, which means that when I merge master into this PR I'll need to remember to add back the relevant bits (I'll get merge conflicts anyway, but just in case).

**Pull request merged**